src/components/events/Event/index.js

```diff
@@ -23,16 +22,25 @@ module.exports = React.createClass({
     };
   },
 
-  // componentWillMount: function() {
-  //   var _this = this;
-  //   var backgroundImage = document.createElement('img');
-  //   backgroundImage.src = this.props.backgroundImageUrl;
-  //   backgroundImage.onload = function() {
-  //     _this.setState({
-  //       blurredImage: effect.blurImage(backgroundImage, 30)
-  //     });
-  //   };
-  // },
+  // () -> ()
+  getInitialState: function() {
+    return {
+      blurredBackgroundImage: ''
+    };
+  },
+
+  // () -> ()
+  componentWillMount: function() {
+    var _this = this;
+    var backgroundImage = document.createElement('img');
+    backgroundImage.setAttribute('crossorigin', 'anonymous');
+    backgroundImage.src = this.props.backgroundImageUrl;
+    backgroundImage.onload = function() {
+      _this.setState({
+        blurredBackgroundImage: effect.blurImage(backgroundImage, 30)
+      });
+    };
+  },
 
   // () -> XJS
```

Comment:
I don't like comments at all*. I added types so I knew what the functions were supposed to do.